From 95d9e3807907d58c809d6605f6860eee0f936087 Mon Sep 17 00:00:00 2001
From: Padmarao Begari <padmarao.begari@microchip.com>
Date: Fri, 22 Oct 2021 12:58:29 +0530
Subject: [PATCH v1 5/5] doc: board: Update Microchip MPFS Icicle Kit doc

UART1 uses for U-BOOT and Linux console instead of UART0 and
UART0 is reserved for Hart Software Services(HSS).

Signed-off-by: Padmarao Begari <padmarao.begari@microchip.com>
---
 doc/board/microchip/mpfs_icicle.rst | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/doc/board/microchip/mpfs_icicle.rst b/doc/board/microchip/mpfs_icicle.rst
index c71c2f3cab..d7af542c0e 100644
--- a/doc/board/microchip/mpfs_icicle.rst
+++ b/doc/board/microchip/mpfs_icicle.rst
@@ -18,8 +18,9 @@ The support for following drivers are already enabled:
 
 1. NS16550 UART Driver.
 2. Microchip Clock Driver.
-3. Cadence MACB ethernet driver for networking support.
-4. Cadence MMC Driver for eMMC/SD support.
+3. Microchip I2C Driver.
+4. Cadence MACB ethernet driver for networking support.
+5. Cadence MMC Driver for eMMC/SD support.
 
 Booting from eMMC using HSS
 ---------------------------
@@ -214,7 +215,8 @@ GPT partition.
 Booting
 ~~~~~~~
 
-You should see the U-Boot prompt on UART0.
+You should see the U-Boot prompt on UART1.
+(Note: UART0 is reserved for HSS)
 
 Sample boot log from MPFS Icicle Kit
 ''''''''''''''''''''''''''''''''''''
@@ -451,7 +453,8 @@ copied payload and Linux image.
 
     sudo dd if=<payload_binary> of=/dev/sdX2 bs=512
 
-You should see the U-Boot prompt on UART0.
+You should see the U-Boot prompt on UART1.
+(Note: UART0 is reserved for HSS)
 
 GUID type
 ~~~~~~~~~
-- 
2.25.1

